core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -3719,3 +3767,19 @@ class ReplicaManagerTest {
     }
   }
 }
+
+class MockReplicaSelector extends ReplicaSelector {
+
+  private val triggerSelectionCount = new AtomicLong()
+
+  def getTriggerSelectionCount: Long = triggerSelectionCount.get
```

Comment:
nit: The `trigger` in the name bugs me. We could perhaps just use `selectionCount`?